activesupport/lib/active_support/notifications/instrumenter.rb

```diff
@@ -137,17 +132,11 @@ def parent_of?(event)
 
       private
         def now
-          Process.clock_gettime(Process::CLOCK_MONOTONIC)
+          Concurrent.monotonic_time
         end
 
-        if clock_gettime_supported?
-          def now_cpu
-            Process.clock_gettime(Process::CLOCK_PROCESS_CPUTIME_ID)
-          end
-        else
-          def now_cpu
-            0
-          end
+        def now_cpu
+          ActiveSupport::ProcessClock.process_time
```

Comment:
Don’t think adding a top level file/constant is needed. Let’s just revert this change.

Reply:
You are right, those changes are unnecessary. It might make sense only if we have at least two cases where we use `Process.clock_gettime(Process::CLOCK_PROCESS_CPUTIME_ID)`.